### PR TITLE
Feat/android fixes

### PR DESCRIPTION
**Pull request by nitesh2599**
Contains gradle updates for: 
  - at_contacts_flutter
  - at_contacts_group_flutter
  - at_events_flutter
  - at_location_flutter
  - at_onboarding_flutter
  - at_sync_ui_flutter
  - at_theme_flutter
  
  https://github.com/atsign-foundation/apps/issues/727#issue-1379116312

### CODE REVIEW

File: at_contacts_flutter/example/pubspec.yaml [hunk 1]
```diff
@@ -39,10 +39,18 @@ dependencies:
...
-  flutter_test:
-    sdk: flutter
+dependency_overrides:
+  biometric_storage: ^4.1.3
+  at_onboarding_flutter:
+    git:
+      url: https://github.com/atsign-foundation/at_widgets.git
+      path: at_onboarding_flutter
+      ref: fix_flutter_qr_reader
+  at_backupkey_flutter:
```
Comment (sachins-geekyants) [inline at_contacts_flutter/example/pubspec.yaml]: should we use hosted reference instead ?

File: at_events_flutter/example/pubspec.yaml [hunk 2]
```diff
@@ -46,7 +46,17 @@ dependencies:
 
 
 dependency_overrides:
-  at_onboarding_flutter: ^5.0.0
+  biometric_storage: ^4.1.3
+  at_onboarding_flutter:
```
Comment (sachins-geekyants) [inline at_events_flutter/example/pubspec.yaml]: Can we use hosted versions ?

File: at_contacts_flutter/example/pubspec.yaml [hunk 3]
```diff
@@ -30,33 +30,28 @@ dependencies:
...
   cupertino_icons: ^1.0.2
   at_app_flutter: 5.0.1
-  at_onboarding_flutter: ^5.0.2
+  at_onboarding_flutter: ^5.0.3
+  at_backupkey_flutter: ^4.0.6
   at_contacts_flutter:
     path: ../
-
-
-dev_dependencies:
```
Comment (sachins-geekyants) [inline at_contacts_flutter/example/pubspec.yaml:43]: why have we removed dev_dependencies ?

File: at_contacts_group_flutter/example/pubspec.yaml [hunk 4]
```diff
@@ -61,6 +58,10 @@ dev_dependencies:
 # For information on the generic Dart part of this file, see the
 # following page: https://dart.dev/tools/pub/pubspec
 
+dependency_overrides:
+  biometric_storage: ^4.1.3
+  at_onboarding_flutter: ^5.0.3
+  at_backupkey_flutter: ^4.0.6
```
Comment (sachins-geekyants) [inline at_contacts_group_flutter/example/pubspec.yaml]: @Pritam-deb can we remove onboarding and backupkey from overrides and move dependencies?

Comment (Pritam-deb) [reply at_contacts_group_flutter/example/pubspec.yaml]: moving them from overrides to dependencies doesn't work.

File: at_contacts_flutter/example/pubspec.yaml (hunk 3 above)
Comment (Pritam-deb) [reply at_contacts_flutter/example/pubspec.yaml:43]: fixed

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**